File: Python/Euler1.py
import time

start = time.clock()

number = 1
sum = 0
while number < 1000:
	if (number % 3 == 0) or (number % 5 == 0):
		sum = sum + number
	number = number + 1

print sum
end = time.clock()
print end - start

File: C_CPlusPlus/Euler1/Euler1.cc
/*
 * Euler1.cc
 *
 *  Created on: Dec 18, 2017
 *      Author: DannyB
 */

#include <iostream>
#include <ctime>
using namespace std;

int main()
{
	clock_t start, stop;
	double elapsed_time;
	unsigned number;
	unsigned sum = 0;

	start = clock();

	for(number = 0; number < 1000; number++)
	{
		if(((number % 3) == 0) ||
		   ((number % 5) == 0))
		{
			sum = sum + number;
		}
	}

	stop = clock() - start;
	elapsed_time = (double(stop) / double(CLOCKS_PER_SEC));

	cout << sum << endl;
	cout << elapsed_time << endl;
}
